Unable to run CI example (permission denied)

I am trying to run the DataKitCI example under `ci/skeleton`. When I try to build the container using the command from the readme, I get the following error:

```
$ docker build -t my-ci --build-arg CONFIG=exampleCI .
Sending build context to Docker daemon  11.26kB
Step 1/9 : FROM datakit/ci
 ---> f16cddfd6f02
Step 2/9 : ARG CONFIG=exampleCI
 ---> Using cache
 ---> c96660dbfaef
Step 3/9 : ADD . /datakit-ci
 ---> Using cache
 ---> 835d5519c636
Step 4/9 : WORKDIR /datakit-ci
 ---> Using cache
 ---> 709a5c6fd874
Step 5/9 : RUN sudo chown opam .
 ---> Using cache
 ---> c083e3152b38
Step 6/9 : RUN opam config exec make $CONFIG && ln _build/skeleton/$CONFIG.native /datakit-ci/datakit-ci && rm -rf _build
 ---> Running in edcdb5c45eaf
ocamlbuild -use-ocamlfind exampleCI.native
Exception Sys_error("/datakit-ci/_build/_log: Permission denied").
make: *** [Makefile:6: exampleCI] Error 100
The command '/bin/sh -c opam config exec make $CONFIG && ln _build/skeleton/$CONFIG.native /datakit-ci/datakit-ci && rm -rf _build' returned a non-zero code: 2
```

What did I miss? Any help is appreciated.

--- a/ci/skeleton/.dockerignore
+++ b/ci/skeleton/.dockerignore
@@ -1,0 +1,2 @@
+*.native
+_build

--- a/ci/skeleton/Dockerfile
+++ b/ci/skeleton/Dockerfile
@@ -4,7 +4,7 @@ ARG CONFIG=exampleCI
 ADD . /datakit-ci
 WORKDIR /datakit-ci
 RUN sudo chown opam .
-RUN opam config exec make $CONFIG && ln _build/skeleton/$CONFIG.native /datakit-ci/datakit-ci && rm -rf _build
+RUN opam config exec make $CONFIG && ln _build/$CONFIG.native /datakit-ci/datakit-ci && rm -rf _build
 USER root
 ENTRYPOINT ["/datakit-ci/datakit-ci"]
 CMD []

